packages/im-base/src/utils/getFormattedTime.js

```diff
@@ -0,0 +1,16 @@
+const getFormattedTime = (seconds) => {
+  const date = new Date(seconds * 1000);
+  const hh = date.getUTCHours();
+  const mm = date.getUTCMinutes();
+  const ss = pad(date.getUTCSeconds());
```

Comment:
should be a more meaningful name

Reply:
which one hours, mins, seconds ? or padding "0"?

Reply:
pad